spec/spec_helper.rb

```diff
@@ -17,39 +10,38 @@ def self.cache
 end
 
 # ActiveRecord::Base.logger = Logger.new(STDOUT)
-ActiveRecord::Base.establish_connection(:adapter => "sqlite3", :database => ":memory:")
+ActiveRecord::Base.establish_connection adapter: 'sqlite3', database: ':memory:'
 # ActiveRecord::Base.configurations = true
 
 ActiveRecord::Schema.verbose = false
-ActiveRecord::Schema.define(:version => 1) do
+ActiveRecord::Schema.define(version: 1) do
   create_table :settings do |t|
-    t.string :var, :null => false
+    t.string :var, null: false
     t.text :value
     t.integer :thing_id
-    t.string :thing_type, :limit => 30
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.string :thing_type, limit: 30
+    t.datetime 'created_at'
+    t.datetime 'updated_at'
   end
 
   create_table :users do |t|
     t.string :login
     t.string :password
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime 'created_at'
+    t.datetime 'updated_at'
   end
 end
 
 RSpec.configure do |config|
-
   config.before(:all) do
-    class ::Setting < RailsSettings::CachedSettings
+    class Setting < RailsSettings::CachedSettings
     end
 
     class User < ActiveRecord::Base
       include RailsSettings::Extend
     end
 
-    ActiveRecord::Base.connection.execute("delete from settings")
+    ActiveRecord::Base.connection.execute('delete from settings')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.